server/controllers/meeting.js

```diff
@@ -7,42 +7,46 @@ const mongoose = require("mongoose");
 // @access Private
 exports.createMeeting = asyncHandler(async (req, res, next) => {
   const { userId, duration } = req.body;
-  const validDuration = ["15", "30", "45"];
-
-  if (
-    !userId ||
-    !duration ||
-    !mongoose.isValidObjectId(userId) ||
-    !validDuration.includes(duration)
-  ) {
+  const validDuration = [15, 30, 45, 60];
+
+  if (!userId || !duration || !validDuration.includes(duration)) {
     res.status(400);
     throw new Error("Invalid meeting data");
   }
 
+  const meetings = await Meeting.find({ userId: userId });
+
+  // Checking is user already has this event
+  for (let i = 0; i < meetings.length; i++) {
+    if (meetings[i].duration === duration) {
```

Comment:
I think we don't want to block users from having multiple 30 minute meetings but rather as mentioned in todays meeting adding a event type url which would be unique.

Reply:
Thanks, I believe we're using the mongo _id as our event type url.